checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
-            "guid": "b1c82a3f-2320-4dfa-8972-7ae4823c8930",
-            "id": "D06.16",
-            "ammp": true,
-            "severity": "High",
-            "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
-            "link": "https://learn.microsoft.com/azure/ddos-protection/ddos-protection-reference-architectures"
-        },
-        {
-            "category": "Network Topology and Connectivity",
-            "subcategory": "Internet",
+            "subcategory": "Firewall",
             "text": "Add diagnostic settings to save logs, using the Resource Specific destination table, for all Azure Firewall deployments.",
             "waf": "Operations",
             "service": "Firewall",
             "guid": "715d833d-4708-4527-90ac-1b142c7045ba",
-            "id": "D06.17",
+            "id": "D07.09",
             "ammp": true,
             "severity": "Medium",
             "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
             "link": "https://learn.microsoft.com/en-us/azure/firewall/firewall-structured-logs"
         },
         {
             "category": "Network Topology and Connectivity",
-            "subcategory": "Internet",
+            "subcategory": "Firewall",
             "text": "Migrate from Azure Firewall Classic rules (if exist) to Firewall Policy.",
             "waf": "Operations",
             "service": "Firewall",
             "guid": "e960fc6b-4ab2-4db6-9609-3745135f9ffa",
-            "id": "D06.18",
+            "id": "D07.10",
             "ammp": true,
             "severity": "Important",
             "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
             "link": "https://learn.microsoft.com/en-us/azure/firewall-manager/migrate-to-policy"
         },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Segmentation",
+            "text": "Use a /26 prefix for your Azure Firewall subnets.",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "22d6419e-b627-4d95-9e7d-019fa759387f",
+            "id": "D07.11",
+            "ammp": true,
+            "severity": "High",
+            "graph": "resources | where type=='microsoft.network/virtualnetworks' | project id,subnets=properties.subnets | mv-expand subnets | project id, subnetName = subnets.name, subnetPrefix = subnets.properties.addressPrefix | extend subnetPrefixLength = split(subnetPrefix, '/')[1] | where subnetName == 'AzureFirewallSubnet' | extend compliant = (subnetPrefixLength == 26) | distinct id, compliant",
+            "link": "https://learn.microsoft.com/azure/firewall/firewall-faq#why-does-azure-firewall-need-a--26-subnet-size"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Optimise Rule Configuration and processing",
```

Comment:
I think this text needs more guidance than just "optimize rule configuration and processing."  It needs to be more explicitly what needs to be done.  The article linked to also doesn't have guidance - on optimal vs. non-optimal rule configuration.